lib/components/atoms/Button/Button.js

```diff
@@ -15,6 +15,7 @@ type Props = {
   secondary?: boolean,
   tertiary?: boolean,
   disabled?: boolean,
+  icon?: boolean,
```

Comment:
Why? This is simply wrong. You are violating SOLID principles here. Specially Single Responsibility and Open Close principles